### ISSUE TITLE
Wrong words extract on wordwise

### ISSUE THREAD

**Issue by py930828** [xxyzz/WordDumb#215]
### Checkboxes

- [X] I have read the [installation](https://xxyzz.github.io/WordDumb/installation.html) and [usage](https://xxyzz.github.io/WordDumb/usage.html) documents.
- [X] I have not found similar open/closed issue or disscussion.
- [X] Reboot doesn't fix the problem.

### Describe the bug and copy the error message

It seems that every word I extract is incorrect. I have unchecked Wiktionary and set Wordwise to use Chinese, which works fine only if I have Wiktionary checked. However, Wiktionary provides fewer word reminders, and most of the words it reminds are not difficult, while it misses a lot of difficult words.

I'm not sure how to make Wordwise function without using Wiktionary. Can you advise me on any alternatives or solutions?

### Plugin settings and reproduce steps

{
  "add_locator_map": false,
  "choose_format_manually": true,
  "cuda": "cu121",
  "fandom": "",
  "kindle_gloss_lang": "zh_cn",
  "minimal_x_ray_count": 1,
  "model_size": "lg",
  "preferred_formats": [
    "AZW3",
    "MOBI",
    "AZW",
    "EPUB",
    "KFX"
  ],
  "python_path": "",
  "remove_link_styles": false,
  "search_people": true,
  "use_all_formats": false,
  "use_gpu": true,
  "use_pos": true,
  "use_wiktionary_for_kindle": false,
  "wiktionary_gloss_lang": "zh_cn",
  "zh_wiki_variant": "cn"
}

### Generated files, screenshots or videos

![Image_20240502034546](https://github.com/xxyzz/WordDumb/assets/25916453/ee0e22d2-0a75-4f3a-8c0c-85072126c8a2)
![Image_20240502034728](https://github.com/xxyzz/WordDumb/assets/25916453/6ef295ec-b581-4b63-aa02-b05cf4f7edbd)

**Comment by xxyzz** [comment]
Delete the klld file in your kindle at `/system/kll/` then connect wifi, kindle will download the klld file again.

**Comment by py930828** [comment]
> Delete the klld file in your kindle at `/system/kll/` then connect wifi, kindle will download the klld file again.

I tried it and it is redownloading the klld file, but it's still giving me the wrong extraction. Might be a bug?

**Comment by xxyzz** [comment]
You also need to disable the "use Wiktionary" option and recreate the Word Wise file to use Kindle's original English to Chinese Word Wise dictionary file.

**Comment by py930828** [comment]
Yeah it is already been disabled, and I have tried to recreate wordwise several times.

**Comment by davidislishuwang** [comment]
It works for me finally
